src/client/cli.py

```diff
@@ -53,14 +97,44 @@ def main(argv=sys.argv):
         dss.initiate(argv)
         return 0
 
-    if argv[1] == '--curl' and len(argv) == 4:
-        common.curlify(argv[2], argv[3])
-    elif argv[1] == '--prettyprint' and len(argv) == 4:
-        common.curlify(argv[2], argv[3], False, True, True)
-    elif len(argv) == 3 and '=' in argv[2]:
-        common.curlify(argv[1], argv[2], False, True)
-    elif len(argv) >= 3:
-        common.curlify(argv[1], argv[2:], True, True)
-    else:
-        print "Invalid client request"
+    if '--debug' in argv:
+        argv.remove('--debug')
+        debug = True
+    elif '-d' in argv:
+        argv.remove('-d')
+        debug = True
+
+    if '--help' in argv or '-h' in argv or 'help' in argv:
+        argv.remove(argv[-1])
+        if argv[-1] == 'iam':
+            for row in iam_help:
+                print row, 'jcs iam ' + row.strip() + ' ' + iam_help[row]
+        else:
+            found = False
+            for row in iam_help:
```

Comment:
in the 'else' part, will this still throw IAM related help? I guess it shouldn't.. Correct me if I am wrong..